client/landing/gutenboarding/components/action-buttons/style.scss

```
@@ -13,6 +13,7 @@
 	justify-content: space-between;
 	display: flex;
 	align-items: center;
+	z-index: z-index( '.block-editor-editor-skeleton__header' );
```

We have our own now 🎉 

https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/_z-index.scss

```suggestion
	z-index: gutenboarding-z-index( '.block-editor-editor-skeleton__header' );
```

---

`.block-editor-editor-skeleton__header` from `@wordpress/base-styles` has a z-index of 30.

I moved them over to `gutenboarding-z-index` and call them `.gutenboarding__header` and `.gutenboarding__footer` with a value of 30 for now. See: https://github.com/Automattic/wp-calypso/pull/42118/commits/ddb970e0ec1883593a5a546bc9034a57a97491c1